xarray/backends/api.py

```diff
@@ -344,12 +345,12 @@ def open_dataset(
         If True, decode the 'coordinates' attribute to identify coordinates in
         the resulting dataset.
     engine : {'netcdf4', 'scipy', 'pydap', 'h5netcdf', 'pynio', 'cfgrib', \
-        'pseudonetcdf'}, optional
+        'pseudonetcdf', 'zarr'}, optional
         Engine to use when reading files. If not provided, the default engine
         is chosen based on available dependencies, with a preference for
         'netcdf4'.
     chunks : int or dict, optional
-        If chunks is provided, it used to load the new dataset into dask
+        If chunks is provided, it is used to load the new dataset into dask
         arrays. ``chunks={}`` loads the dataset with dask using a single
         chunk for all arrays.
```

Comment:
Wonder if it's a good idea to put backend specific notes here:

```suggestion
        If chunks is provided, it is used to load the new dataset into dask
        arrays. ``chunks={}`` loads the dataset with dask using a single
        chunk for all arrays. When using ``engine="zarr"`, if `chunks='auto'`,
        dask chunks are created based on the variable's zarr chunks, and if
        `chunks=None`, zarr array data will lazily convert to numpy arrays upon
        access. This accepts all the chunk specifications as Dask does.
```